transactions/reset-epoch/2022/aug-25/README.md

```diff
@@ -0,0 +1,123 @@
+# Reset Epoch
+
+> Aug 25th, 2022
+
+## New method using flow cli and multi-sign site
+
+1. DapperLabs generates the Signature Request ID on the [site](https://flow-multisig-git-service-account-onflow.vercel.app/mainnet?type=serviceAccount&name=reset_epoch_with_end_staking_auction.cdc&param=&acct=0x8624b52f9ddcd04a&limit=9999) for the `reset_epoch_with_end_staking_auction.cdc` transaction with the given args.
```

Comment:
This is missing the payload, so remember to paste it or add it here for quickness.